docs/user/login-alternatives.md

```diff
@@ -1,33 +1,11 @@
 # Kubeapps Login
 
-By default, when first visiting Kubeapps, a login form is shown for the user to introduce a Kubernetes API token:
+By default, Kubeapps allow you to login using a Kubernetes service account token. While this is enabled by default to make it easier to get started, the **recommended way of authenticate in Kubeapps is using an OAuth2/OIDC provider**. Depending on your cluster provider, the instructions may differ. Check this [guide](./using-an-OIDC-provider.md) for more information.
+
+When using the service account authentication, the login form is shown for the user to introduce a Kubernetes API token:
 
 ![Dashboard Login](../img/dashboard-login.png)
 
 The goal of the login form is to identify the user and associate it with a Kubernetes service account. This identity information will be used by Kubeapps to authenticate the user against the Kubernetes API. You can find more information about access control in Kubeapps in this [document](./access-control.md).
 
 However, it's possible to disable the form or delegate the authentication to an OAuth2/OIDC provider so users don't need to introduce a token in the login form.
```

Comment:
I think it's better to remove the part `... it's possible to disable the form ... `  too.

Reply:
Good catch, thanks @batiati I missed that.